modules/cross-origin-destination/module.php

```diff
@@ -0,0 +1,54 @@
+<?php
+namespace Elementor\Modules\CrossOriginDestination;
+
+use Elementor\Core\Experiments\Manager as Experiments_Manager;
+use Elementor\Core\Utils\Collection;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+class Module extends \Elementor\Core\Base\Module {
+
+	public function get_name() {
+		return 'cross-origin-destination';
+	}
+
+	public function __construct() {
+		parent::__construct();
+
+		$this->add_filters();
+	}
+
+	protected function add_filters() {
+		add_filter( 'elementor/element/link-attributes', function ( $attributes ) {
+			$rel = empty( $attributes['rel'] ) ? '' : $attributes['rel'];
+
+			$rel_attributes = new Collection( preg_split(
+				'/\s+/',
+				$rel,
+				-1,
+				PREG_SPLIT_NO_EMPTY
```

Comment:
Can you do some normal split on space? (like normal people 😉 😆 )

Reply:
Done